cdap-docs/reference-manual/source/cli-api.rst

```diff
@@ -62,12 +62,12 @@ The executable should bring you into a shell, with a prompt similar to:
 
 This indicates that the CLI is currently set to interact with the CDAP server at ``localhost``.
 
-- To interact with a different CDAP server by default, set the environment variable ``CDAP_HOST`` to a hostname.
+- To interact with a different CDAP server by default, run the CLI by specifying the uri ``-u example.com``.
```

Comment:
suggestion: "To interact with a remote CDAP server, run ...".  ("by default" no longer appropriate)